src/Command/BenchmarkCommand.php

```diff
@@ -46,24 +47,39 @@ protected function configure()
             ->setName('benchmark')
             ->addOption('iteration', 'i', InputArgument::OPTIONAL, 'Number of iteration(s)', 1)
```

Comment:
the `InputArgument` for the rest of the options is wrong, right?

Reply:
Indeed, but as the value of the constants are the same, it works :)
A PR to fix this would been appreciated.